source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -127,7 +127,11 @@ def get_queryset(self):
         return User.objects.filter()
 
     def get(self, request, *args, **kwargs):
-        if self.request.user.has_perm('apiVolontaria.get_user'):
+        if 'profile' in self.kwargs.keys():
+            self.kwargs['pk'] = self.request.user.id
+            return self.retrieve(request, *args, **kwargs)
+
+        elif self.request.user.has_perm('apiVolontaria.get_user'):
             return self.retrieve(request, *args, **kwargs)
```

Comment:
So having the permission `get_user` allows you to get ANY user?

Reply:
Yep! Permissions are always for all objects, we can't associate a permissions with a specific object (or we need to write some logic)